cli/command/environment.go

```diff
@@ -78,17 +73,13 @@ func (e *EnvironmentCommand) Command() cli.Command {
 			},
 			{
 				Name:      "set-scale",
-				Usage:     "update the min/max scale of an environment cluster",
+				Usage:     "update the scale of a static environment cluster",
 				Action:    e.setScale,
 				ArgsUsage: "ENVIRONMENT_NAME",
```

Comment:
This should be `ENVIRONMENT_NAME SCALE` since the scale is a required arg, and flags denote optional params.

Reply:
✅